.github/workflows/coffee.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   coffee-break:
     runs-on: ubuntu-latest
     container:
-      image: golang:1.20
+      image: quay.io/redhat-appstudio-qe/qe-tools:latest
```

Comment:
```bash
❯ docker images | grep quay.io/redhat-appstudio-qe/qe-tools
quay.io/redhat-appstudio-qe/qe-tools     latest           631de591014a   9 hours ago     2.39GB
```
I think the image is too big, IMO the image should contain only the binary and `config` directory and could be based on a tiny image like `registry.access.redhat.com/ubi8/ubi-minimal`

Reply:
Since we are building binary in go used ubi9/go-toolset:1.20.

Reply:
The final image size would be 1.17 GB, and the base image is around 1.11 GB.

Reply:
The thing is, there's no need to use the base builder image in the final image. It's required only for building the binary. For the final base image, as I mentioned above, `ubi-minimal` will be sufficient

[example](https://github.com/redhat-appstudio/build-service/blob/d524220d6c8e4de134300ae4b395eb44e849132b/Dockerfile#L22-L26)

Reply:
#112